### PR TITLE
Create .deepsource.toml to find errors in shell and python

### PR DESCRIPTION
**Pull request by MrGithub2021**
This will help us all catch errors.

### CODE REVIEW

**Comment by patricklodder** [comment]
Do you have an example report ran against your repository you can share?

**Comment by MrGithub2021** [comment]
Yes, I have it [here](https://deepsource.io/gh/MrGithub2021/dogecoin/).

**Review by Technosword — APPROVED** [review]

File: .deepsource.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version = 1
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```
Comment (MrGithub2021) [inline .deepsource.toml:8]: python

File: .deepsource.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+[[analyzers]]
+name = "test-coverage"
+enabled = true
+
+[[analyzers]]
+name = "shell"
+enabled = true
```
Comment (MrGithub2021) [inline .deepsource.toml:16]: Coverage and shell

**Comment by MrGithub2021** [comment]
................

**Pull request closed without merging**